py/picca/tests/delta_extraction/expected_flux_tests.py

```diff
@@ -144,37 +145,50 @@ def test_dr16_expected_flux_compute_continuum(self):
                 continue
             cols = line.split()
             los_id = int(cols[0])
-            continuum = np.array([float(item) for item in cols[1:]])
+            if len(cols) == 1:
+                continuum = None
+            else:
+                continuum = np.array([float(item) for item in cols[1:]])
             continua[los_id] = continuum
         f.close()
 
         # compare the results
         correct_forests = 0
         for forest in data.forests:
-            if not np.allclose(forest.continuum, continua.get(forest.los_id)):
-                print("Difference found in forest.continuum")
-                print(f"forest.los_id: {forest.los_id}")
-                print(f"result test are_close result-test")
-                for i1, i2 in zip(forest.continuum, continua.get(forest.los_id)):
-                    print(i1, i2, np.isclose(i1, i2), i1-i2)
-            self.assertTrue(
-                np.allclose(forest.continuum, continua.get(forest.los_id)))
+            if forest.continuum is None:
+                if continua.get(forest.los_id) is not None:
+                    print(f"For forest with los_id {forest.los_id}, new continuum "
+                          "is None. Expected continua:")
+                    print(continua.get(forest.los_id))
+                self.assertTrue(continua.get(forest.los_id) is None)
+            elif continua.get(forest.los_id) is None:
+                self.assertTrue(forest.continuum is None)
```

Comment:
this assertTrue will always fail by definition (if it wouldn't we'd be in the if-clause above). The check is still reasonable to do (we could also just raise an AssertionError), but needs a similar msg as above.